Example/Core/Tests/GTMHTTPServer.m

```diff
@@ -0,0 +1,631 @@
+/* Copyright (c) 2010 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//
+//  Based a little on HTTPServer, part of the CocoaHTTPServer sample code
+//  http://developer.apple.com/samplecode/CocoaHTTPServer/index.html
```

Comment:
linke doesn't work - what's the licensing?

Reply:
Seems to be a BSD of some kind https://opensource.apple.com/source/HTTPServer/HTTPServer-11/OpenSourceLicenses/CocoaHTTPServer.txt.auto.html